src/ErrorHandling.h

```diff
@@ -169,6 +169,8 @@ typedef enum
     SUBCODE_CONFIGURATION_FAIL_MP_NICDATA1,
     SUBCODE_FAIL_MP_NICDATA_INIT1,
     SUBCODE_FAIL_INVALID_BASE_TRACK_MOTION_TYPE,
+    SUBCODE_FAIL_INIT_SERVICE_LIST_JOBS,
+    SUBCODE_FAIL_ADD_SERVICE_LIST_JOBS,
```

Comment:
I need to check why CI didn't flag these as undocumented.

Reply:
This isn't getting flagged as there is a *catch-all* section for `8011` ([here](https://github.com/Yaskawa-Global/motoros2/blob/f61a7238d78066aa565542b6fefd112bad42d0cb/doc/troubleshooting.md#alarm-8011xx)), and CI isn't setup to complain about those (as otherwise it would always complain, seeing as we're 'relying' on these catch-alls).

These `SUBCODE_FAIL_INIT_*` and `SUBCODE_FAIL_ADD_*` subcodes are all in `8011`, hence no errors raised.

Adding `--no-catch-alls` to the linter's configuration results in:

```
src/ErrorHandling.h:94:5: error: no documentation for '8010[0]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:95:5: error: no documentation for '8010[1]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:97:5: error: no documentation for '8010[3]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:102:5: error: no documentation for '8011[0]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:103:5: error: no documentation for '8011[1]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:104:5: error: no documentation for '8011[2]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:105:5: error: no documentation for '8011[3]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:106:5: error: no documentation for '8011[4]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:107:5: error: no documentation for '8011[5]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:108:5: error: no documentation for '8011[6]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:109:5: error: no documentation for '8011[7]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:110:5: error: no documentation for '8011[8]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:111:5: error: no documentation for '8011[9]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:113:5: error: no documentation for '8011[10]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:114:5: error: no documentation for '8011[11]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:115:5: error: no documentation for '8011[12]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:116:5: error: no documentation for '8011[13]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:117:5: error: no documentation for '8011[14]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:121:5: error: no documentation for '8011[18]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:172:5: error: no documentation for '8011[64]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:173:5: error: no documentation for '8011[65]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:179:5: error: no documentation for '8012[0]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:180:5: error: no documentation for '8012[1]' in 'doc/troubleshooting.md'
src/ErrorHandling.h:181:5: error: no documentation for '8012[2]' in 'doc/troubleshooting.md'
```

Reply:
So technically, these new subcodes are already documented. Just like many others in this category, by the `8011[xx]` catch-all.

Would that be sufficient for you @ted-miller?

Reply:
Yeah, I forgot about the [xx]